examples/inference-client/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We should use Apache 2.0 since that's what the rest of the repo uses. No need to create unnecessary ambiguity between two licenses that are almost the same.

Reply:
Done.